Formula/gr-osmosdr.rb

```diff
@@ -0,0 +1,24 @@
+class GrOsmosdr < Formula
+  desc "GNU Radio block for interfacing with various radio hardware"
+  homepage "https://osmocom.org/projects/gr-osmosdr/wiki/GrOsmoSDR"
+  url "https://github.com/osmocom/gr-osmosdr/archive/refs/tags/v0.2.4.tar.gz"
+  sha256 "28b6f2768aee7b397b227e9e70822e28de3b4c5362a5d14646a0948a48094a63"
+  license "GPL-3.0-or-later"
+
+  depends_on "boost" => :build
+  depends_on "cmake" => :build
+  depends_on "gnuradio"
+  depends_on "libsndfile"
+  depends_on "pybind11"
+  depends_on "soapysdr"
+
+  def install
+    system "cmake", "-S", ".", "-B", "build", *std_cmake_args, "-DENABLE_NONFREE=TRUE"
+    system "cmake", "--build", "build"
+    system "cmake", "--install", "build"
+  end
+
+  test do
+    system "false"
```

Comment:
Can you propose a more substantial test than checking the version or calling for help? These tests rarely ever fail even for broken installations.

>We want tests that don’t require any user input and test the basic functionality of the application. For example `foo build-foo input.foo` is a good test and (despite their widespread use) `foo --version` and `foo --help` are bad tests.

Ref. https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula

Reply:
@chenrui333 I've added a little test to make sure the python imports. The issue is that the interface basically needs hardware to connect to.